packages/material-ui-styles/src/makeStyles/makeStyles.d.ts

```diff
@@ -70,7 +69,11 @@ export type StylesHook<S extends Styles<any, any>> = StylesRequireProps<S> exten
   ? (props?: any) => ClassNameMap<ClassKeyOfStyles<S>>
   : (props: PropsOfStyles<S>) => ClassNameMap<ClassKeyOfStyles<S>>;
 
-export default function makeStyles<S extends Styles<any, any>>(
-  styles: S,
-  options?: WithStylesOptions<ThemeOfStyles<S>>,
-): StylesHook<S>;
+export default function makeStyles<
+  Theme = unknown,
```

Comment:
@eps1lon One last thing though; is `unknown` too strict? What if some people prefer to use untyped theme? i.e., could it just be `any` by default, or does that break any conventions?

Reply:
The philosophy is that people can opt-in to loose typings but have strict types by default. Going from loose to strict is not possible in most scenarios. 

Or rather I want to be as close as possible to DefinitelyTyped as possibles. As far as I know unknown types are actually typed as `unknown` not as `any`. At least for types that support TS ^3.0